### DISCUSSION TITLE
Lua scripting

### DISCUSSION THREAD

**Discussion by jmarsik** [BehaviorTree/BehaviorTree.CPP#611] [Ideas]
Hello, just wondering if you have completely abandoned the idea of using Lua interpreter (basics of intergration described on https://www.lua.org/pil/24.1.html) for scripting inside behavior tree definition? You wrote about it on Discourse here https://discourse.behaviortree.dev/t/first-step-toward-lua-scripting/43 but nobody answered. And then you implemented your own scripting language that can be used in behavior tree definition inside XML.

Do you still plan to integrate Lua?

Would that be something that one can contribute or do you think that it's perhaps a too complex dependency or any other reason against it?

**Comment by facontidavide** [comment]
Yes, the idea is temporarily abandoned.

If you want to contribute, please be my guess!

The place where I see a good fit is to implement your custom Actions / Conditions / Decorators without C++.

If you have a prove of concept and you want to work together, let me know, I will help, but not lead this effort
